docs/developer/checkout.mdx

```diff
@@ -380,6 +380,33 @@ mutation {
 }
 ```
 
+## Stock reservations
+
+While stock availability is checked on every step of the checkout process no guarantee is made to the client that stock will not be sold out before checkout completion. In case when item with limited stock was added to multiple checkouts, it will be allocated to first completed checkout causing remaining checkouts to fail to advance to the next step even if they were created before the completed one, with the `INSUFFICIENT_STOCK` error being returned by the API.
+
+Saleor implements optional stock reservations feature where stock will be eagerly allocated to the checkout for configured time.
+
+### Enabling reservations
+
+To enable reservations, navigate to "Site settings" page in the dashboard. Here, in the "Checkout Configuration" section you'll find controls allowing you to enable stock reservations for authenticated and anonymous checkouts.
```

Comment:
For this, we should update the dashboard docs and link them here. I'll open a task for that. @karolkielecki Will you be able to handle this? @rafalp Maybe we should describe which API fields control these settings and link to API reference describing these fields/mutations?

Reply:
@maarcingebala Yes, you can assign it to me. Please provide all the details in the task.